### PR TITLE
D.1.8 View List of Active Mentorships (contract)

### PR DESCRIPTION
**Pull request by natalieloke**
# Feature

Use Case: D.1.8 View List of Active Mentorships 
Feature ID : 11
Feature: Mentorship Tools
Figma link for wireframe: 
IMG_0626 2

# Changelog:
- see commit log

Note: progress value of 50 is a placeholder first

## Checklist:

- [x] Merged latest develop
- [x] PR title makes sense

## Screenshots (where relevant):
<a href="https://www.loom.com/share/8f38a1912dcb44bfaae4a473bb304fde">
    <p>Digi Dojo | View List of Active Mentorship Contracts - Watch Video</p>
    
  </a>

Screenshot 2021-04-12 at 5 10 06 PM

### CODE REVIEW

File: src/pages/student/dashboard/mentorships/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+  }
+
+  useEffect(() => {
+    getOngoingMentorshipContracts()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  const viewMentorshipContract = id => {
+    const path = `/student/mentorship/subscription/${id}`
```
Comment (rawrrencee) [inline src/pages/student/dashboard/mentorships/index.js]: do you intend for the dashboard submenu to not be shown here? coz right now it only checks for /student/dashboard

**Review by rawrrencee — DISMISSED** [review]
looks good to me

**Review by y1bs — DISMISSED** [review]
Looks good!

File: src/pages/student/dashboard/mentorships/index.js (hunk 1 above)
Comment (natalieloke) [reply src/pages/student/dashboard/mentorships/index.js]: updated file path to include /dashboard
Screenshot 2021-04-12 at 5 52 15 PM

**Review by rawrrencee — APPROVED** [review]
wokies nice

**Pull request merged**